common/src/main/res/values/styles.xml

```diff
@@ -1121,4 +1121,10 @@
         <item name="backgroundColor">@android:color/transparent</item>
         <item name="strokeColor">@android:color/transparent</item>
     </style>
+
+    <style name="RoundedTextView">
+        <item name="cornerRadius">6dp</item>
+        <item name="backgroundColor">@color/system_green_0.1</item>
+        <item name="strokeColor">@color/system_green_0.1</item>
```

Comment:
Thanks, I checked our master branch and we have this Style, which is doing the same thing, but with a `5dp` corner instead of `6dp`.
```
    <style name="GreenTextHighlight">
        <item name="backgroundColor">@color/system_green_0.1</item>
        <item name="strokeColor">@android:color/transparent</item>
        <item name="cornerRadius">5dp</item>
    </style>
    ```
    
Should we only have one?

Reply:
Yeah I think 5dp is fine.